plugin/pkg/global/resourcereferencemanager/admission_test.go

```diff
@@ -198,7 +198,7 @@ var _ = Describe("resourcereferencemanager", func() {
 		Context("tests for SecretBinding objects", func() {
 			It("should accept because all referenced objects have been found (secret found in cache)", func() {
 				kubeInformerFactory.Core().V1().Secrets().Informer().GetStore().Add(&secret)
```

Comment:
Is there a reason why you didn't wrap the `Add()` calls to `kubeInformerFactory` in `Expect`s like the others?

Reply:
I just missed them. Thanks for pointing out :)